src/status_im/contexts/wallet/connected_dapps/view.cljs

```diff
@@ -1,29 +1,129 @@
+                                                     :account (:name wallet-account)})}]))}]))
+
+(defn- on-dapp-disconnect-press
+  [wallet-account dapp]
+  (rf/dispatch [:show-bottom-sheet
+                {:content (fn [] [disconnect-dapp/view
+                                  {:customization-color (:color wallet-account)
+                                   :dapp                dapp
+                                   :on-disconnect       #(on-disconnect wallet-account dapp)}])}]))
+
+(defn- account-details
+  [{:keys [name emoji color]}]
+  (let [theme (quo.theme/use-theme)]
+    [rn/view {:style style/account-details-wrapper}
+     [quo/context-tag
+      {:theme               theme
+       :type                :account
+       :size                24
+       :account-name        name
+       :emoji               emoji
+       :customization-color color}]]))
+
 (defn- header
-  [{:keys [title subtitle]}]
-  [:<>
-   [rn/view {:style style/header-container}
-    [quo/button
-     {:icon-only?          true
-      :type                :grey
-      :background          :blur
-      :size                32
-      :accessibility-label :close-scan-qr-code
-      :on-press            #(rf/dispatch [:navigate-back])}
-     :i/close]]
-   [quo/text
-    {:size   :heading-1
-     :weight :semi-bold
-     :style  (style/header-text (when subtitle true))}
-    title]])
+  [{:keys [title wallet-account on-close on-add]}]
+  (let [{:keys [color]} wallet-account]
+    [:<>
+     [rn/view {:style style/header-container}
+      [quo/button
+       {:icon-only?          true
+        :type                :grey
+        :background          :blur
+        :size                32
+        :accessibility-label :connected-dapps-close
+        :on-press            on-close}
+       :i/close]]
+     [rn/view {:style style/header-wrapper}
+      [quo/standard-title
+       {:title               title
+        :accessibility-label :connected-dapps
+        :customization-color color
+        :right               [plus-button/plus-button
+                              {:on-press            on-add
+                               :accessibility-label :connected-dapps-add
+                               :customization-color color}]}]
+      [rn/view {:flex 1}
+       [account-details wallet-account]]]]))
 
 (defn view
   []
-  [rn/view {:style {:flex 1}}
-   [header {:title "Connected dApps"}]])
+  (let [{:keys [bottom]}                   (safe-area/get-insets)
+        {:keys [color] :as wallet-account} (rf/sub [:wallet/current-viewing-account])
+        pairings                           (rf/sub [:wallet-connect/pairings])
+        theme                              (quo.theme/use-theme)]
+    [rn/view {:flex 1}
+     [header
+      {:title          (i18n/label :t/connected-dapps)
+       :wallet-account wallet-account
+       :on-close       #(rf/dispatch [:navigate-back])
+       :on-add         #(js/alert "Feature not implemented")}]
```

Comment:
There is a `not-implemented` namespace with this alert as a function. 

It's better to use that so in future we can do a global lookup of functions that need to be implemented.

Reply:
You're right, for features that won't be implemented for a while we must use the same function for easier discoverability. Here though it won't stay like this for long, as `on-add` will be implemented in the coming weeks anyways and without needing a lookup, so should be fine for now. WDYT?